data/dfndata/npc/hymenoptera_base.dfn

```diff
@@ -0,0 +1,373 @@
+// Base hymenoptera with shared DFN tags
+[base_hymenoptera]
+{
+DIRECTION=RND
+BACKPACK
+NPCWANDER=4
+FX2=20
+NPCAI=2
+FLAG=NEUTRAL
+}
+
+// ambitious black solen queen - Base NPC with shared DFN tags
+[base_ambitious_black_solen_queen]
+{//quest npc
+NAME=an ambitious black solen queen
+ID=0x30F
+BACKPACK
+SKIN=0x453
+RACE=31
+NPCWANDER=0
+ORIGIN=lbr
+}
+
+[ambitious_black_solen_queen]
+{
+GETUO=ambitious_black_solen_queen_lbr
```

Comment:
So the implication is that this LBR+ based on the name.

Should there be a GETUO, GETT2A, GETUOR or GETTD if they're prior?  What happens if they're missing?